.pre-commit-config.yaml

```diff
@@ -6,51 +6,57 @@ repos:
   rev: v4.4.0
   hooks:
   - id: check-yaml
-    args: [--unsafe]
   - id: check-json
   - id: end-of-file-fixer
   - id: trailing-whitespace
   - id: check-case-conflict
 
--   repo: https://github.com/dbt-labs/pre-commit-hooks
-    rev: v0.1.0a1
-    hooks:
-    -   id: dbt-core-in-adapters-check
+- repo: https://github.com/macisamuele/language-formatters-pre-commit-hooks
+  rev: v2.14.0
+  hooks:
+  - id: pretty-format-yaml
+    args: [--autofix, --indent, '2', --offset, '2', --preserve-quotes]
```

Comment:
We should either make this a long form list (like `black`'s `args`) or make other lists short form. I think one of the formatters also had an option for enforcing that automatically.